package.json

```diff
@@ -123,6 +123,49 @@
         "command": "io.orta.jest.coverage.toggle",
         "title": "Jest: Toggle Coverage Overlay"
       }
+    ],
+    "debuggers": [
+      {
+        "type": "node",
+        "label": "Debug Jest tests using vscode-jest"
+      },
```

Comment:
don't think we need this dummy one anymore...

Reply:
I initially thought that it would be necessary when registering a `CustomDebugProvider`, but it turned out it isn't, I therefore removed it.